ci/ray_ci/doc/cmd_check_api_discrepancy.py

```diff
@@ -1,23 +1,88 @@
 import click
+from typing import Dict, Set
 
 from ci.ray_ci.doc.module import Module
+from ci.ray_ci.doc.autodoc import Autodoc
+from ci.ray_ci.doc.api import API
+
+TEAM_API_CONFIGS = {
+    "data": {
+        "head_modules": {"ray.data", "ray.data.grouped_data"},
```

Comment:
Why do we include `ray.data.grouped_data` in particular and not another module like `ray.data.read_api`?